contributing/documentation/standards.rst

```diff
@@ -57,6 +57,8 @@ Code Examples
   Unless the example requires a custom bundle, make sure to always use the
   ``AppBundle`` bundle to store your code;
 * Use ``Acme`` when the code requires a vendor name;
+* Use ``example.com`` as the domain of sample URLs and ``example.org`` when a
+  second domain is required. Both domains are `reserved by the IANA`_.
```

Comment:
I've updated the link to the RFC. However, I don't understand your first suggestion. This document is about standards, so everything is "an order" (i.e. "strong suggestion") to the doc contributor.